src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/GlobalSearchResource.kt

```diff
@@ -76,8 +76,8 @@ class GlobalSearchResource(
       PageRequest.of(0, 10)
     )
     val mid = System.currentTimeMillis()
-    val totalNomisNumber = getTotalNomisNumber()
     val totalIndexNumber = prisonerIndexService.countIndex(indexStatusService.getCurrentIndex().currentIndex)
+    val totalNomisNumber = getTotalNomisNumber()
```

Comment:
This should fix the problem of the index sometimes being > nomis. I think that prisoners can be created during the slow nomis query, so getting the index first means it is more likely to equal the nomis count (which will be that at the _start_ of the nomis query execution)